src/lib.rs

```diff
@@ -68,16 +61,31 @@ impl<AccountId, Moment> Shipment<AccountId, Moment> {
     }
 }
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug)]
+pub enum ShippingOperation {
+    Pickup,
```

Comment:
```suggestion
    Pickup,
    InTransit,
```

For a package/cargo delivery, usually it will be checked and scanned at multiple points before finally reaching the destination. Will be good if we add an intermediate state `InTransit`. See [the story I am working on](https://github.com/gautamdhameja/substrate-enterprise-sample/pull/11).

Reply:
I'd call differently than the Shipment's InTransit status. Maybe a generic term like 'Scan' ?